<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PyTranslator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>trans_icon.png</normaloff>trans_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_Translation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>380</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>380</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>380</width>
      <height>70</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>배달의민족 도현</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>번역</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Original_Box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>380</width>
      <height>200</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>380</width>
      <height>200</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>380</width>
      <height>200</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
   </widget>
   <widget class="QComboBox" name="Original_language">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="Trans_language">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Trans_Box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>380</width>
      <height>200</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>380</width>
      <height>200</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>380</width>
      <height>200</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>20</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
